KardiaTeleOp.java

```diff
@@ -93,12 +96,26 @@ public void loop() {
         lift.manualUpdate(gamepad2);
         effector.manualUpdate(gamepad2);
 
+        // Drivetrain throttle mode switch
+        if (gamepad1.start && !is_start_pressed) {
```

Comment:
This whole code section can't be here.  I think init only runs once.  This should be in the `loop()`, if I understand correctly